app/views/api/v2/split_registries/show.json.jbuilder

```diff
@@ -0,0 +1,10 @@
+json.splits do
+  @split_registry.splits.each do |split|
+    json.set! split.name do
+      json.weights split.registry
+      json.feature_gate split.feature_gate?
```

Comment:
Another advantage of this representation would be that we could theoretically set different sample rates for different splits if e.g. different apps were running at different orders of magnitude of scale.